PyDSS/cli/run.py

```diff
@@ -91,4 +98,17 @@ def run(project_path, options=None, tar_project=False, zip_project=False, verbos
             print(f"options must be of type dict; received {type(options)}")
             sys.exit(1)
 
-    PyDssProject.run_project(project_path, options=options, tar_project=tar_project, zip_project=zip_project)
+    project = PyDssProject.load_project(project_path, options=options)
+    project.run(tar_project=tar_project, zip_project=zip_project, dry_run=dry_run)
+
+    if dry_run:
+        print("="*37)
+        maxlen = max([len(k) for k in project.estimated_space.keys()])
+        if len("ScenarioName") > maxlen:
+            maxlen = len("ScenarioName")
+        template = "{:<{width}}   {}\n".format("ScenarioName", "EstimatedSpace (bytes)", width=maxlen)
+        for k, v in project.estimated_space.items():
+            template += "{:<{width}} : {}\n".format(k, str(v), width=maxlen)
```

Comment:
1. Can you print the total of all scenarios?
2. Can you make the number of bytes human readable?  A value of 100 GiB expressed in bytes in hard to interpret.  There are plenty of simple implementations online.

Reply:
Here is a possible idea for the compression problem.
- We know the total possible size without compression.
- After we close the h5py.File we can look at the size of what actually go written.
- We could divide those two numbers to get a compression ratio.
- We could print that compression ratio and an estimate for the total - which would assume the ratio applies to the entire dataset, which may or may not be true.

Reply:
Added into tutorial after our discussion and test.